grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcService.java

```diff
@@ -72,6 +73,8 @@
 class UnframedGrpcService extends SimpleDecoratingService<HttpRequest, HttpResponse>
         implements ServiceWithPathMappings<HttpRequest, HttpResponse> {
 
+    private static final char LINE_SEPARATOR = '\n';
```

Comment:
Sounds fine. While after change to use `System.lineSeparator()`, shall we also change the constant '\n' to `System.lineSeparator()` in the unit test? Otherwise, the unit test will fail in the local of developers who are using windows?

Reply:
I think it's OK to use `\n` here for consistency of server responses. i.e. the server that runs on Windows should give the same response with the server that runs on *nix.

Reply:
Got it. Btw there is no need to revert, I didn't change this part in the latest update commit :)

Reply:
Ah, OK. :-)